### PR TITLE
get Glare to compile on latest version of Gleam

### PR DESCRIPTION
**Pull request by ajstrand**
My PR gets glare to compile on the latest version of gleam, since there was a lot of syntax changes needed.

### CODE REVIEW

File: example/demo/gleam.toml [hunk 1]
```diff
@@ -9,6 +9,9 @@ description = "A Gleam project"
...
 # links = [{ title = "Website", href = "https://gleam.run" }]
 
+
+target = 'javascript'
+
 [dependencies]
-gleam_stdlib = "~> 0.29"
-glare = "~> 0.1"
+gleam_stdlib = "~> 0.31"
+glare = {path = "/Users/alex/Documents/projects/glare"}
```
Comment (Nicd) [inline example/demo/gleam.toml]: Probably not what you want to merge.

Comment (ajstrand) [reply example/demo/gleam.toml]: I was using the glare library as a local dependency. Im not sure how to abstract getting the path to the library in something like a .toml file. Maybe theres a library i can use to do that.

Comment (ajstrand) [reply example/demo/gleam.toml]: You cant use variables inside .toml files, so ill have to set this value to a placeholder for a new commit.

Comment (Enderchief) [inline example/demo/gleam.toml]:
```suggestion
glare = {path = "../../"}
```

**Review by Enderchief — CHANGES_REQUESTED** [review]
Thanks so much. Sorry for the late review.

**Pull request merged**